app/src/main/res/values/styles.xml

```diff
@@ -57,4 +57,8 @@
         <item name="android:background">?selectableItemBackground</item>
         <item name="android:gravity">center_vertical</item>
     </style>
+
+    <style name="ExpandedToolbarTextStyle">
+        <item name="android:textSize">26sp</item>
```

Comment:
Hi @ydasc815 can you please add this hardcoded value in dimens.xml file and the use refer it or alternatively you can use the value which is near to this 26sp value from the already present values of dimens file.

Reply:
Yes, sure. I'll update the requested changes. Apart from that, please refer the PR Description section. I have already attached the screenshots relevant to this issue.

Reply:
Hi @ydasc815 i know you have added the screenshots but i just them to be added under **How this has been tested section ** with more interactive way . if you don't know how then take a look at this pr --> https://github.com/anitab-org/mentorship-android/pull/725 [here either, you can use a gif]  or use screenshots like this --> https://github.com/anitab-org/mentorship-android/pull/722